config/200-jsm-dispatcher-service.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: jetstream-ch-dispatcher
+  namespace: knative-eventing
+  labels:
+    nats.eventing.knative.dev/release: devel
```

Comment:
I think this should be natsjsm?
Just like here:
https://github.com/knative-sandbox/eventing-natss/pull/182/files#diff-a9bd98b0e5248a4625de7f3a112db792e398a7fd49924ac9bd40bec7f0a788fdR20

Reply:
@vaikas Or we should be `nats`（include natss and  natsjsm）? We public both to together. What do you think about?